tests/props/host/host.c

```diff
@@ -126,7 +126,7 @@ int main(int argc, const char* argv[])
 {
     oe_result_t result;
     oe_enclave_t* enclave = NULL;
-    bool is_signed = false;
+    int test_type = 0;
```

Comment:
For clarity, can you use named enumerations here instead of raw ints?
```c
typedef enum
{
    TEST_PROPS_UNSIGNED,
    TEST_PROPS_SIGNED,
    TEST_PROPS_CUSTOM_SIGNED
} test_type_t;
```
```suggestion
    test_type_t test_type;
```